contents/index.md

```diff
@@ -15,7 +15,7 @@
 </style>
 
 <div class="language-switcher">
-<a href="/"><button>🇺🇸</button></a>
+<a href="https://juliadatascience.io"><button>🇺🇸</button></a>
 <a href="/pt"><button>🇧🇷</button></a>
```

Comment:
```suggestion
<a href="https://juliadatascience.io/pt"><button>🇧🇷</button></a>
```
What about this?
If someone spams the portuguese icon it will go into a recursive `pt/pt/pt/pt/pt` path.